src/python/gudhi/representations/metrics.py

```diff
@@ -12,6 +12,7 @@
 from sklearn.metrics import pairwise_distances
 from gudhi.hera import wasserstein_distance as hera_wasserstein_distance
 from .preprocessing import Padding
+from joblib import Parallel, delayed
```

Comment:
I wanted to ask to add joblib as a `:Requires:` in the documentation, but even if it is not clearly said in scikit-learn documentation, it seams joblib is installed when you install scikit-learn. So I am ok if it is not explicitly written in the documentation.

Reply:
Exactly. I was going to say that in the PR, and then completely forgot :disappointed: